pp_sketch/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 '''PopPUNK sketching functions'''
 
-__version__ = '1.7.4'
+__version__ = '1.7.6.2'
```

Comment:
v2 PR has been pending one final fix (this only changes the CLI options, and will make the python API more robust by making everything support a kwarg, but not change it).

Anyway, I will sort out that merge after this, as that's still some time away from being ready (#67)